pianos.geojson

```diff
@@ -84,6 +84,20 @@
           35.890820
         ]
       }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "description": "Piano in the airport",
```

Comment:
Is it in the gates area or outside? 😄 

Thanks!

Reply:
In the gates area but I don't remember the number of the gate 🔍